tv_apps/smart-system/js/preview_window.js

```diff
@@ -90,10 +93,45 @@
     }
   };
 
+  PreviewWindow.prototype._handle_mozbrowsercontextmenu = function(evt) {
```

Comment:
After offline discussion, we decided to do not move this part to AppInstallManager since AppModalDialog will lost the support of focusManager.focus()